apps/dapp-console-api/src/testhelpers/auth.ts

```diff
@@ -1,21 +1,16 @@
-import type { PrivyClient } from '@privy-io/server-auth'
 import type { AnyRouter } from '@trpc/server'
 import { createCallerFactory } from '@trpc/server'
+import bcrypt from 'bcrypt'
 import type { Request, Response } from 'express'
 import type { getIronSession } from 'iron-session'
-import { type Mock, vi } from 'vitest'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this prevents the following type error: 
<img width="663" alt="Screenshot 2024-03-28 at 3 50 35 PM" src="https://github.com/ethereum-optimism/ecosystem/assets/15234569/48b92640-e885-4b7e-aa95-20dacd1de7e2">